crates/astria-conductor/src/executor/optimism.rs

```diff
@@ -0,0 +1,111 @@
+//! This module contains functionality for watching for OP-Stack L1 deposit events
+//! and converting them to deposit transactions to be executed on a rollup based on
+//! the OP-Stack.
+//!
+//! It works by querying for deposit events from the `OptimismPortal` contract.
```

Comment:
You can link `OptimismPortal` by writing [`OptimismPortal`] If it's in scope, then rust will link to its docs.